src/hooks/useSyncNetworkParamWithStore.ts

```diff
@@ -70,8 +35,7 @@ export function useSyncNetworkParamWithStore() {
           isOnInit.current = false
           return
         }
-        if ((possibleWallet || possibleWallet === null) && !account) {
-          // connected wallet but web3-react slow return account, or checking connection
+        if (!account) {
```

Comment:
active network is eth
enter https://kyberswap-interface-1954.pr.kyberengineering.io/swap/bnb => not change change => bug

Reply:
remove if (!account)

Reply:
fixed